example/poisson/poisson_package.cpp

```diff
@@ -86,6 +86,20 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
     pkg->AddParam("sparse_accessor", sp_accessor);
   }
 
+  // ParArrays for reductions We stash the array, and AllReduce
+  // object, which is per MPI rank, in the pkg object.
+  // We must use a mutable Param here so that
+  // the object can be modified by the driver.
+  parthenon::AllReduce<HostArray1D> view_reduce;
+  view_reduce.val = HostArray1D("Reduce me", 10);
+  for (int i = 0; i < view_reduce.val.size(); i++) {
+    view_reduce.val(i) = 0;
+  }
+  for (int i = 0; i < view_reduce.val.size(); i++) {
+    view_reduce.val(i) += i;
+  }
```

Comment:
Is the first `for` required or just there to illustrate "initialization"?

Reply:
Just to illustrate initialization. I will make a comment saying so.